### DISCUSSION TITLE
testWithdrawMultiplefunderscheaper error!!!

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2676] [Q&A] [answered]
![Screenshot (65)](https://github.com/user-attachments/assets/f49a25eb-9b75-4659-bf9b-34c8702ca367)

**Comment by EngrPips** [comment, marked as answer]
Hello @prakharmishra3956, It says `assertion failed` so I will advise you do one assert at a time so you know which assert is failing.

**Reply by prakharmishra3956** [reply]
Thanks, it's done by removing all the assert.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
